task_sdk/src/airflow/sdk/execution_time/supervisor.py

```diff
@@ -297,6 +298,10 @@ def start(
         if pid == 0:
             # Parent ends of the sockets are closed by the OS as they are set as non-inheritable
 
+            del path
+            del client
+            del ti
```

Comment:
Why delete this? want to remove accidental use?

Reply:
Likely not required, but just to make 100% sure they get gc'd and don't leak around longer than they are needed

Reply:
Worth adding a comment?